conn/connManager.go

```diff
@@ -182,22 +187,66 @@ func Do(req *http.Request) (*http.Response, error) {
 
 	quicConf := &quic.Config{
 		Tracer: opsTracer,
+		MaxIdleTimeout: 500 *time.Millisecond,
 	}
 
 	getRoundTripper().TLSClientConfig = tlsConfig
 	getRoundTripper().QuicConfig = quicConf
 
-	//defer roundTripper.Close()
+	elapsed := time.Since(start).Seconds()
+	connLogger.Info("Connection setup time for requesting:", "setup_time", elapsed)
 
-	client = &http.Client{
-		Transport: httplog.LoggingRoundTripper{Base: getRoundTripper()},
+	start =  time.Now()
+	epAddrs, err := GetAllEpAddresses(req.URL.Hostname())
+	elapsed = time.Since(start).Seconds()
+	connLogger.Info("DNS lookup time for requesting:", "dns_lookup_time", elapsed)
```

Comment:
Don't need the `:` of the string `DNS lookup time for requesting:`

Reply:
Indeed.